packages/api-explorer/src/index.jsx

```diff
@@ -210,13 +217,14 @@ class ApiExplorer extends React.Component {
                       accordion
                       onChange={this.props.onDocChange}
                     >
-                      {this.props.docs.map((doc, index) => (
-                        <Panel 
+                      {this.props.docs.map(doc => (
+                        <Panel
                           header={this.renderHeaderPanel(doc)}
+                          key={doc._id}
                           style={{...styleByMethod(doc.api.method), ...panelStyle}}
-                          key={index}
                           forceRender={this.props.forcePanelRender}
                         >
+                          <Waypoint onEnter={() => this.waypointEntered(doc._id)} fireOnRapidScroll={false} />
```

Comment:
Why fireOnRapidScroll set to false? We should render some endpoint after the end of the page

Reply:
if at the loading of the component we have an "anchor" set on the last child, we will trigger hundreds of items together

Reply:
removed waypoint